nominees/surveillance-forms.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Surveillance Forms",
+  "description": "Online forms for reporting rumored COVID-19 cases.",
+  "website": "http://forms.ethiopia-covid19.com/",
```

Comment:
@318anushka the `websiteURL` seems to be a broken link, kindly update it with a valid one. Thanks.

Reply:
@nathanbaleeta projects are still in progress so website(mentioned here) isn't available yet. I'll change it to [websiteURL](https://ethiocovid19rt.com/).